frontend/src/pages/Overview/OverviewTable.js

```diff
@@ -146,139 +156,43 @@ const getTableEntries = ({
-                  showProjectAdditionalData(id);
-                }}
-                title="Additional Data"
-                icon={<MoreIcon />}
-                data-test={`project-overview-additionaldata-${index}`}
-              />
-              <ActionButton
-                notVisible={!canViewPermissions}
-                onClick={() => showProjectPermissions(id)}
-                title={strings.common.show_permissions}
-                icon={<PermissionIcon />}
-                data-test={`pp-button-${index}`}
-                iconButtonStyle={styles.editIcon}
-              />
-              <ActionButton
-                notVisible={!isOpen && editDisabled}
-                onClick={() => {
-                  showEditDialog(id, displayName, description, thumbnail, projectedBudgets, tags);
-                }}
-                title={strings.common.edit}
-                icon={<EditIcon />}
-                data-test={`pe-button`}
-                iconButtonStyle={styles.editIcon}
-              />
-            </div>
-          </List>
-        </CardContent>
-      </Card>
+      <ProjectCard
+        key={index}
+        index={index}
+        id={id}
+        allowedIntents={allowedIntents}
+        closeSearchBar={closeSearchBar}
+        history={history}
+        displayName={displayName}
+        mappedStatus={mappedStatus}
+        projectedBudgets={projectedBudgets}
+        budgets={budgets}
+        displayedTags={displayedTags}
+        dateString={dateString}
+        showProjectAdditionalData={showProjectAdditionalData}
+        additionalDataEmpty={additionalDataEmpty}
+        canViewPermissions={canViewPermissions}
+        isOpen={isOpen}
+        showProjectPermissions={showProjectPermissions}
+        editDisabled={editDisabled}
+        showEditDialog={showEditDialog}
+        description={description}
+        thumbnail={thumbnail}
+        tags={tags}
+        classes={classes}
+        imagePath={imagePath}
+      />
     );
   });
 };
 
 const OverviewTable = props => {
   const tableEntries = getTableEntries(props);
   return (
-    <div
-      aria-label="projects"
-      style={{
-        backgroundColor: "transparent",
-        height: "100%",
-        width: "100%",
-        display: "flex",
-        flexDirection: "row",
-        flexWrap: "wrap",
-        flex: 1,
-        alignItems: "center",
-        justifyContent: "center"
-      }}
-    >
+    <div aria-label="projects" style={styles.tableEntries}>
       {tableEntries}
-      <Card data-test="projectcreation" style={{ height: "510px", margin: "35px", width: "300px", opacity: "0.7" }}>
-        <div
-          style={{
-            display: "flex",
-            backgroundColor: "lightgray",
-            flexDirection: "row",
-            maxWidth: "350px",
-            height: "100%",
-            alignItems: "center",
-            justifyContent: "center"
-          }}
-        >
+      <Card data-test="projectcreation" style={styles.addProject}>
```

Comment:
```suggestion
<Card data-test="project-creation" style={styles.addProject}>
```

all of our data-test tags are written seperated by hyphens

Reply:
changed to "project-creation"